app/src/main/java/org/xbmc/kore/ui/NowPlayingFragment.java

```diff
@@ -311,18 +321,7 @@ public void onVolumeMuteClicked(View v) {
         action.execute(hostManager.getConnection(), new ApiCallback<Boolean>() {
             @Override
             public void onSuccess(Boolean result) {
-                if (!isAdded()) return;
-                if (result) {
-                    Resources.Theme theme = getActivity().getTheme();
-                    TypedArray styledAttributes = theme.obtainStyledAttributes(new int[] {
-                            R.attr.colorAccent});
-                    volumeMuteButton.setColorFilter(
-                            styledAttributes.getColor(0,
-                                    getActivity().getResources().getColor(R.color.accent_default)));
-                    styledAttributes.recycle();
-                } else {
-                    volumeMuteButton.clearColorFilter();
-                }
+                setVolumeMuteButton(!result);
```

Comment:
Hmmm, this is working the opposite for me (which is consistent with the previous comment).
When you click the mute button on Kore it mutes Kodi, but in Kore's interface it appears as not muted (the button isn't highlighted). Maybe you were assuming that if the button is highlighted it means that it's not muted but it's the other way around.

Looking into this i was wondering if this check (the call to application.GetProperties to check the muted status) should be done in the HostConnectionObserver, so that it gets done regularly, to always keep Kore in sync with Kodi (this PR only keeps it in sync in the `OnResume` call). But it's one more call to Kodi (there are 3 already) being done regularly (if connecting through HTTP, through TCP we needed to implement the `Application.OnVolumeChanged` which isn't being acted upon right now), so i'm not sure if it should be done... People might have noticed that the interface isn't in sync but i haven't heard any complains so it might not be worth doing it regularly. 
Right now, doing it in the `OnResume` handler looks ok.